src/legacy/models/userHelpers.js

```diff
@@ -165,6 +166,7 @@ function getPlainUser(user, { orgUnitsField }, toArray) {
         userGroups: namesFromCollection(user.userGroups, "displayName", toArray),
         organisationUnits: namesFromCollection(user.organisationUnits, orgUnitsField, toArray),
         dataViewOrganisationUnits: namesFromCollection(user.dataViewOrganisationUnits, orgUnitsField, toArray),
+        teiSearchOrganisationsUnits: namesFromCollection(user.dataViewOrganisationUnits, orgUnitsField, toArray),
```

Comment:
We can keep the entity name at the object level, but the dhis2 property name to get the value:

```tsx
searchOrganisationsUnits: namesFromCollection(user.teiSearchOrganisationUnits, orgUnitsField, toArray),
```